### DISCUSSION TITLE
Search result page is blank beyond 50+ pagination

### DISCUSSION THREAD

**Discussion by YogeshT15** [microsoft-search/pnp-modern-search#1791] [Q&A]
**Version: 4.2.3**

Result web-part work fine till 50 page, on click of 51 (or +) page result web part looks completely blank. 

If I can show 5 items per page then till 250 item (5 * 50) its search works fine.  
If I can show 10 items per page then till 500 item (10 * 50) its search works fine.  

is there any paging limitation?

**Comment by FranckyC** [comment]
https://docs.microsoft.com/en-us/sharepoint/search-limits could explain the 500 number ("Number of rows in a result set"). But wait you are displaying more than 500 items per page? If yes this is not a best practice for sure.

**Reply by wobba** [reply]
Moved to discussion as issue is missing important details such as provider and repro setup. But seems an API issue, so you should try to repro it using another approach on the API you are using. If it works outside the web parts then compare your request with what the web parts is issuing and pinpoint the issue.

**Comment by YogeshT15** [comment]
@FranckyC 

Results are visible till 500 items loading . After 500 item , result webpart does not responding. 

**Example 1:** consider result count is 1000. per page item configured 10. in this case till page 50 i.e. till result item 500 result will be visible. After that result webpart doesn't work. Entire page looks blank.

**Example 2:** consider result count is 10000. per page item configure as 100. in this case till page 5 i.e. till result item 500 (100 * 50) result will be visible. After that result webpart doesn't work. Entire page looks blank. 

By considering this behavior, its looks like result webpart have 500 items limitation.
